src/pages/dashboard/dashboard.tsx

```diff
@@ -59,80 +77,83 @@ const Dashboard = () => {
-                  <p className={globalStyles.textXLarge} data-cy="unstaked">
-                    {data ? formatAndRoundApi3(data.userUnstaked) : UNKNOWN_NUMBER}
-                  </p>
-                </div>
-              </>
-            }
-            footer={
-              // TODO: In case there is a pending unstake there should be no button, just green arrow (see figma)
-              <Button type="link" onClick={() => setOpenModal('unstake')} disabled={!canInitiateUnstake}>
+              </div>
+            </CardHeader>
+          }
+          content={
+            <div className={styles.cardContent}>
+              <div>
+                <p className={styles.cardContentTitle}>total</p>
+                <p className={styles.cardContentValue} data-cy="balance">
+                  {tokenBalances ? formatAndRoundApi3(tokenBalances.userTotal) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+              <div>
+                <p className={styles.cardContentTitle}>withdrawable</p>
+                <p className={styles.cardContentValue} data-cy="withdrawable">
+                  {tokenBalances ? formatAndRoundApi3(tokenBalances.withdrawable) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+            </div>
+          }
+          contentFooter={
+            <Button type="secondary" size="sm" onClick={() => setOpenModal('withdraw')} disabled={!canWithdraw}>
+              Withdraw
+            </Button>
+          }
+        />
+
+        <Card
+          header={
+            <CardHeader>
+              <h5>Staking</h5>
+              <div className={styles.cardHeaderButton}>
+                <Tooltip overlay="You need to deposit API3 tokens before staking">
+                  <img src={images.helpOutline} alt="new proposal help" />
+                </Tooltip>
+                <Button type="secondary" size="sm" onClick={() => setOpenModal('stake')} disabled={!canStake}>
+                  Stake
+                </Button>
+              </div>
+            </CardHeader>
+          }
+          content={
+            <div className={styles.cardContent}>
+              <div>
+                <p className={styles.cardContentTitle}>staked</p>
+                <p className={styles.cardContentValue} data-cy="staked">
+                  {data ? formatAndRoundApi3(data.userStaked) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+              <div>
+                <p className={styles.cardContentTitle}>unstaked</p>
+                <p className={styles.cardContentValue} data-cy="unstaked">
+                  {data ? formatAndRoundApi3(data.userUnstaked) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+            </div>
+          }
+          contentFooter={
+            !pendingUnstake && (
+              <Button size="sm" type="secondary" onClick={() => setOpenModal('unstake')} disabled={!canInitiateUnstake}>
```

Comment:
Currently we use Figma's "Secondary gradient" as secondary button. These action buttons (Withdraw, Initiate Unstake...) should be "Secondary neutral" variant. Can you add that variant?

Reply:
Also, the button size should be `xs` not `sm`

Reply:
Done. I also created the `/variants` folder to hold the styles separately. I would like to move your recent file `link-styles.module.scss` in there too, you're okay with that?

![image](https://github.com/user-attachments/assets/4401951d-88e8-445b-bd2f-ad6b45f08800)

Reply:
Good job, it's definitely better this way. Go ahead and move the link styles there as well 👍